### PR TITLE
chore: CI files update

### PR DESCRIPTION
**Pull request by bonddim**
- Fix chart publish
- Update Dockerfile
- Export binaries from docker images
- Create release with assets from workflow
- Remove circleci config

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -40,14 +41,20 @@ RUN CGO_ENABLED=0 GOARCH=${TARGETARCH} go build -o /build/dist/wg-portal \
...
 # Final image
 ######
 FROM alpine:3.19
 # Install OS-level dependencies
 RUN apk add --no-cache bash curl iptables nftables openresolv
 # Setup timezone
-ENV TZ=Europe/Vienna
+ENV TZ=UTC
 # Copy binaries
 COPY --from=builder /build/dist/wg-portal /app/wg-portal
```
Comment (h44z) [inline Dockerfile:59]: what is the improvement of using another build stage here?

and if there is some kind of improvement, shouldnt that be: --from=binaries ?

**Review by h44z — CHANGES_REQUESTED** [review]

File: Dockerfile (hunk 1 above)
Comment (h44z) [reply Dockerfile:59]: ah nvm - just seen the change in docker-publish.yml

**Comment by bonddim** [comment]
@h44z, 
Could you disable CircleCI integration completely or at least for tags and master branch?
With this change, workflow will create release on GitHub and upload binaries into.

**Comment by h44z** [comment]
should be removed I guess

**Pull request merged**